src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="block-stars stars--0">
```

Comment:
missed requirement, this block must has class stars stars--0
<img width="849" alt="Screenshot 2023-02-16 at 16 59 10" src="https://user-images.githubusercontent.com/53228259/219400382-9dbc8147-003b-4234-ad6c-72d973a02a76.png">